src/css/displayvideo.css

```diff
@@ -1,11 +1,18 @@
+.main-sidebar-flex {
+  display: flex;
+  justify-content: space-around;
+  margin-top: 1rem;
+}
 .video-container {
   width: 780px;
   height: 400px;
-  border: 1px solid black;
+  /* border: 1px solid black; */
+  border-bottom: 1px solid #d1d5db;
   display: flex;
   flex-flow: column;
```

Comment:
can create a css utility class for flex,since it is used at most of the places. Just a suggestion